sdk/go.mod

```diff
@@ -2,34 +2,44 @@ module github.com/hashicorp/vault/sdk
 
 go 1.13
 
+replace github.com/hashicorp/vault/api => ../api
+
+replace github.com/hashicorp/vault => ../
```

Comment:
Won't this create a dep cycle, since vault imports the sdk, now the sdk imports vault?

Reply:
Oddly enough this was the only way I could get the tests to run in `stepwise_test.go`. Otherwise I get errors about a `helper/random` package. I'll continue to poke around and try to sort things out.

Reply:
Resolved this by moving the `logicaltest` package back to `/helpers` , so the SDK no longer depends on Vault